packages/volto/cypress/support/commands.js

```diff
@@ -819,8 +819,8 @@ Cypress.Commands.add(
   },
 );
 
-Cypress.Commands.add('getSlateEditorAndType', (type) => {
-  cy.getSlate().focus().click().type(type);
+Cypress.Commands.add('getSlateEditorAndType', (text) => {
+  cy.getSlate().should('be.visible').focus().click().type(text);
 });
 
 Cypress.Commands.add('getSlateEditorSelectorAndType', (selector, type) => {
```

Comment:
Should we make the same change for getSlateEditorSelectorAndType?

Reply:
@davisagli to be sure, I did this in `getSlate`.